app/services/profile_fields/add_coding_fields.rb

```diff
@@ -0,0 +1,21 @@
+module ProfileFields
+  class AddCodingFields
```

Comment:
Given some of these are very community-specific I really don't think these belong in the codebase. 

I understand that adding fields from scratch might be tedious but I think a better solution would be a bulk import feature using a CSV or something like that and then in our Forem documentation or playbooks we offer different CSVs that have fields tailored to different communities. For example one CSV might have technical fields for a tech community, another might be geared to Forems for corporations, another for pastime Forems.

What are your thoughts on that?

Reply:
Just to clarify, where you mostly talking about the coding related fields? Here's how I currently think about the presets:

* base fields we want for every community, so I call them in the installer.
* The work-related fields seem generic enough that other communities may want them.
* Links are already a bit debatable since they are heavily skewed towards development and design.
* The coding fields are not particularly useful for anyone, I mentioned that above.

Let me give a CSV importer a shot, that's usually very easy in Ruby.

Reply:
Ok, I have a nicely working basic CSV importer now, will add some more tests around it. Thanks for the idea @mstruve!